common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -173,7 +173,7 @@ class BulkOperationsMixin(object):
     mongo_connection.
     """
     def __init__(self, *args, **kwargs):
-        super(BulkOperationsMixin, self).__init__(*args, **kwargs)
+        super(BulkOperationsMixin, self).__init__()
```

Comment:
1) How did this ever work? 2) Now we are accepting args, kwargs, but ignoring them, which doesn't seem good.  Is it possible that multiple inheritance is making this trickier than it looks?